src/main/java/org/dcsa/ovs/notifications/model/MailConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package org.dcsa.ovs.notifications.model;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import java.util.Map;
+
+@Data
+@ConfigurationProperties(prefix = "dcsa.email")
+@Component
+public class MailConfiguration {

```

Comment:
Should we make the config immutable?
Example:  https://www.baeldung.com/configuration-properties-in-spring-boot#immutable-configurationproperties-binding